teamscale-gradle-plugin/src/main/kotlin/com/teamscale/TeamscaleUploadTask.kt

```diff
@@ -52,9 +61,4 @@ open class TeamscaleUploadTask : DefaultTask() {
             }
```

Comment:
It won't because the expection is rethrown below, but I wrapped it into a GradleExpection now. TBH I can't remember why I added the error logging here.